### DISCUSSION TITLE
502 Bad Gateway (ngnix 1.18.0) Ubuntu

### DISCUSSION THREAD

**Discussion by egp130** [nautobot/nautobot#4628] [Q&A]
Got this message after i install the onboading device plugin inside the nautobot server. Followed the documentation guide in nautobot, no errors after the post_upgrade. I can still access to the cli but not the gui.  Is there anyone experience this error message and how do you resolve this issue?

**Comment by gsnider2195** [comment]
Do you have any logs from the nautobot server itself? 502 bad gateway from nginx sounds like the nautobot server failed to start and nginx can't communicate with it.

**Reply by travisjohnsonga** [reply]
Agreed - that's usually when nautobot service failed to start.

sudo systemctl status nautobot 

What does that say?

**Comment by DoesIThavePower** [comment]
Was this ever resolved?  I started getting this error after upgrading from 2.2 to 2.4.
